src/ppl/test/large_groups3.ok

```diff
@@ -12,13 +12,10 @@
 Found 0 macro blocks.
 [WARNING PPL-0092] Pin group of size 300 does not fit any section. Adding to fallback mode.
 [INFO PPL-0100] Group of size 300 placed during fallback mode.
-[INFO PPL-0010] Tentative 0 to set up sections.
-[INFO PPL-0001] Number of slots          1754
-[INFO PPL-0002] Number of I/O            400
-[INFO PPL-0003] Number of I/O w/sink     400
-[INFO PPL-0004] Number of I/O w/o sink   0
-[INFO PPL-0005] Slots per section        200
-[INFO PPL-0006] Slots increase factor    0.01
-[INFO PPL-0008] Successfully assigned pins to sections.
-[INFO PPL-0012] I/O nets HPWL: 27153.80 um.
-No differences found.
+[WARNING PPL-0078] Not enough available positions (36) in section (47.215, 100.8)-(14.915, 100.8) at edge TOP to place the pin group of size 71.
+[WARNING PPL-0078] Not enough available positions (35) in section (85.215, 100.8)-(47.405, 100.8) at edge TOP to place the pin group of size 71.
+[WARNING PPL-0042] Unsuccessfully assigned I/O groups.
+[WARNING PPL-0097] The max contiguous slots (35) is smaller than the group size (71).
+[WARNING PPL-0097] The max contiguous slots (35) is smaller than the group size (71).
+[ERROR PPL-0093] Pin group of size 71 does not fit in the constrained region 0-100.13 at TOP edge. First pin of the group is pin300.
```

Comment:
This used to produce "Successfully assigned pins to sections" but now fails. Is this intended?

Reply:
Yes. In this case, placing the large group in the middle of the constrained region results in not enough room for the other group constrained to the same region.

In this case, I decided to generate an error instead of placing the group outside the constrained area.